Makefile.in

```diff
@@ -347,7 +347,7 @@ endif
 $(srcdir)/%/.git:
 	cd $(srcdir) && \
 	flock `git rev-parse --git-dir`/config git submodule init $(dir $@) && \
-	flock `git rev-parse --git-dir`/config git submodule update --progress $(dir $@)
+	flock `git rev-parse --git-dir`/config git -c "http.version=HTTP/1.1" submodule update --progress --depth 1 $(dir $@)
```

Comment:
`-c "http.version=HTTP/1.1" forces http 1.1 to avoid error which was seen in previous PR.